scipy/stats/_multivariate.py

```diff
@@ -3850,3 +3852,455 @@ def rvs(self, dim, size=1, random_state=None):
+
+    The probability mass function for `multivariate_hypergeom` is
+
+    .. math::
+
+        P(X_1 = x_1, X_2 = x_2, \ldots, X_k = x_k) = \frac{\binom{m_1}{x_1}
+        \binom{m_2}{x_2} \cdots \binom{m_k}{x_k}}{\binom{M}{n}}, \\ \quad
+        (x_1, x_2, \ldots, x_k) \in \mathbb{N}^k \text{ with }
+        \sum_{i=1}^k x_i = n
+
+    where :math:`m_i` are the number of objects of type :math:`i`, :math:`M`
+    is the total number of objects in the population (sum of all the
+    :math:`m_i`), and :math:`n` is the size of the sample to be taken
+    from the population.
+
+    .. versionadded:: 1.6.0
+
+    Examples
+    --------
+
+    To evaluate the probability mass function of the multivariate
+    hypergeometric distribution, with a dichotomous population of size
+    :math:`10` and :math:`20`, at a sample of size :math:`12` with
+    :math:`8` objects of the first type and :math:`4` objects of the
+    second type, use:
+
+    >>> from scipy.stats import multivariate_hypergeom
+    >>> multivariate_hypergeom.pmf(x=[8, 4], m=[10, 20], n=12)
+    0.0025207176631464523
+
+    The `multivariate_hypergeom` distribution is identical to the
+    corresponding `hypergeom` distribution (tiny numerical differences
+    notwithstanding) when only two types (good and bad) of objects
+    are present in the population as in the example above. Consider
+    another example for a comparison with the hypergeometric distribution:
+
+    >>> from scipy.stats import hypergeom
+    >>> multivariate_hypergeom.pmf(x=[3, 1], m=[10, 5], n=4)
+    0.4395604395604395
+    >>> hypergeom.pmf(k=3, M=15, n=4, N=10)
+    0.43956043956044005
+
+    The functions ``pmf``, ``logpmf``, ``mean``, ``var``, ``cov``, and ``rvs``
+    support broadcasting, under the convention that the vector parameters
+    (``x``, ``m``, and ``n``) are interpreted as if each row along the last
+    axis is a single object. For instance:
+
+    >>> multivariate_hypergeom.pmf(x=[[8, 4], [3, 1]], m=[[10, 20], [10, 5]], n=[12, 4])
+    array([0.00252072, 0.43956044])
+
+    This combines the result of previous two examples in a single array.
+    In this example, the last axis of the each array ``x``, ``m``, and ``n``
+    are treated as independent objects and the ``pmf`` is exaluated over
+    each of them to generate a combined result.
+
+    Here, ``x.shape == (2, 2)``, ``m.shape == (2, 2)``, and
+    ``n.shape == (2,)`` but following the rules mentioned above, they behave
+    as if the rows ``[8, 4]`` and ``[3, 1]`` in ``x``, ``[10, 20]`` and
+    ``[10, 5]`` in ``m``, and ``12`` and ``4`` in ``n`` were two independent
+    objects, and as if we had two populations with ``x.shape = (2,)``,
+    ``m.shape = (2,)``, and ``n.shape = ()``. To obtain the individual
+    elements without broadcasting,  we would do this:
+
+    >>> multivariate_hypergeom.pmf([8, 4], m=[10, 20], n=12)
+    0.0025207176631464523
+    >>> multivariate_hypergeom.pmf([3, 1], m=[10, 5], n=4)
+    0.4395604395604395
```

Comment:
```suggestion
```
What do you think - do all these words help explain what is going on, or do they just make it more confusing?